packages/cdk/resources/RestApiGateway.ts

```diff
@@ -81,12 +76,11 @@ export class RestApiGateway extends Construct {
       managedPolicies: []
     })
 
-    /* Dummy Method/Resource to allow a gateway to be deployed "empty" */
-    apiGateway.root.addMethod("ANY", new MockIntegration({
-      integrationResponses: [
-        {statusCode: "418"}
-      ]
-    }))
+    const authorizer = new CognitoUserPoolsAuthorizer(this, "Authorizer", {
+      authorizerName: "cognitoAuth",
+      cognitoUserPools: [props.userPool],
+      identitySource: "method.request.header.authorization"
```

Comment:
I think this is the default, we could be explicit but I think I would prefer specify the minimum we can

Reply:
https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_apigateway.CognitoUserPoolsAuthorizer.html#identitysource reads as though it is required

Reply:
identitySource?
Type: string (optional, default: IdentitySource.header('Authorization'))